tests/numerics/project_to_surface_01.output

```diff
@@ -11,8 +11,8 @@ DEAL::    Line 1, projected point=3.000 1.500  (line is from 3.000 1.000 to 3.00
 DEAL::    Line 2, projected point=1.500 1.000  (line is from 1.000 1.000 to 3.000 1.000)
 DEAL::    Line 3, projected point=1.500 3.000  (line is from 1.000 3.000 to 3.000 3.000)
 DEAL::  Case 1
-DEAL::    Line 0, projected point=0.707 0.707  (line is from 0.000 1.414 to -1.414 2.828)
-DEAL::    Line 1, projected point=2.121 2.121  (line is from 1.414 2.828 to 0.000 4.243)
+DEAL::    Line 0, projected point=0.000 1.414  (line is from 0.000 1.414 to -1.414 2.828)
+DEAL::    Line 1, projected point=1.414 2.828  (line is from 1.414 2.828 to 0.000 4.243)
```

Comment:
have you verified that these differences are expected?

Reply:
Yes. In this case the old implementation was wrong: the old projected point was not on the line segment, but it was the closest point on the line.